org.deskflow.deskflow.yml

```diff
@@ -0,0 +1,107 @@
+app-id: org.deskflow.deskflow
+runtime: org.kde.Platform
+runtime-version: "6.8"
+sdk: org.kde.Sdk
+command: deskflow
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=network
+  - --device=dri
+  - --talk-name=org.a11y.*
+  - --talk-name=org.kde.StatusNotifierWatcher
+
+modules:
+  - name: python3-attrs
+    buildsystem: simple
+    build-commands:
+      - pip3 install --verbose --exists-action=i --no-index --find-links="file://${PWD}"
+          --prefix=${FLATPAK_DEST} --no-build-isolation attrs
+    sources:
+      - type: file
+        url: https://files.pythonhosted.org/packages/e0/44/827b2a91a5816512fcaf3cc4ebc465ccd5d598c45cefa6703fcf4a79018f/attrs-23.2.0-py3-none-any.whl
+        sha256: 99b87a485a5820b23b879f04c2305b44b951b502fd64be915879d77a7e8fc6f1
+    cleanup:
+      - '*'
+  - name: python3-Jinja2
+    buildsystem: simple
+    build-commands:
+      - pip3 install --verbose --exists-action=i --no-index --find-links="file://${PWD}"
+        --prefix=${FLATPAK_DEST} --no-build-isolation Jinja2
+    sources:
+      - type: file
+        url: https://files.pythonhosted.org/packages/30/6d/6de6be2d02603ab56e72997708809e8a5b0fbfee080735109b40a3564843/Jinja2-3.1.3-py3-none-any.whl
+        sha256: 7d6d50dd97d52cbc355597bd845fabfbac3f551e1f99619e39a35ce8c370b5fa
+      - type: file
+        url: https://files.pythonhosted.org/packages/87/5b/aae44c6655f3801e81aa3eef09dbbf012431987ba564d7231722f68df02d/MarkupSafe-2.1.5.tar.gz
+        sha256: d283d37a890ba4c1ae73ffadf8046435c76e7bc2247bbb63c00bd1a709c6544b
+    cleanup:
+      - '*'
+  - name: libei
+    buildsystem: meson
+    config-opts:
+        - -Ddocumentation=[]
+        - -Dtests=disabled
+        - -Dliboeffis=disabled
+    sources:
+        - type: git
+          url: https://gitlab.freedesktop.org/libinput/libei
+          tag: 1.3.0
+          commit: 997b7c0f37faea4f8bae59613c8f27370925d5b0
+  - name: libportal
+    buildsystem: meson
+    config-opts:
+      - -Ddocs=false
+      - -Dtests=false
+      - -Dportal-tests=false
+      - -Dvapi=false
+      - -Dbackend-qt5=auto
+      - -Dbackend-qt6=auto
+      - -Dbackend-gtk4=auto
```

Comment:
```suggestion
      - -Dbackend-qt6=auto
```

Reply:
only qt6 ? done

Reply:
where are the others ?

*hint* not in this runtime. Only gtk3. 

But that's about what the app uses anyway.